src/App.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Switch, Route } from 'react-router-dom';
+import Settings from './pages/Settings/settings';
```

Comment:
Renomeando o arquivo do componente Settings para index, poderíamos trabalhar dessa forma:
```suggestion
import Settings from './pages/Settings';
```